doc/response-tagging.rst

```diff
@@ -1,17 +1,17 @@
-Extra Invalidation Handlers
-===========================
+Response Tagging
+================
 
-This library provides decorators that build on top of the ``CacheInvalidator``
-to simplify common operations.
+This library provides a service to handle tagging of responses ``ResponseTagger``
+to simplify use of tags.
 
 .. _tags:
 
-Tag Handler
------------
+Response Tagger
+---------------
 
-The tag handler helps you to mark responses with tags that you can later use to
-invalidate all cache entries with that tag. Tag invalidation works only with a
-``CacheInvalidator`` that supports ``CacheInvalidator::INVALIDATE``.
+The ResponseTagger helps you to mark responses with tags that you can later use to
+invalidate all cache entries with that tag. Response Tagging and tag invalidation
+require a proxy client that implements ``TagsInterface``.
```

Comment:
i think this introduction is enough, the first line after the main title is saying kind of the same